libqtile/window.py

```diff
@@ -602,6 +629,22 @@ def _items(self, name):
     def _select(self, name, sel):
         return None
 
+    def to_relative_top(self, **kwargs):
```

Comment:
Why "relative"? :)

Reply:
That's a leftover from before, where the windows were separated into "normal", "floating" and "fullscreen" before any z-values were taken into account. Back then the windows were only moved to the top of their cohort.
It is probably time to rename this to something else now - and think about `cmd_bring_to_front`, where this method could be called instead of doing other things.

Reply:
I added a short docstring.

Reply:
I would be fine with that, too.

Reply:
I removed this and some other stuff in an attempt to shift responsibility for the layering to the layouts (a little).